README.md

```diff
@@ -20,24 +20,90 @@ Specification for binding services to runtime applications running in Kubernetes
 
 Main section of the doc.  Has sub-section that outline the design.
 
-### 1.  Making a service bindable
+### Making a service bindable
 
-*  What metadata is needed for a service to be bindable (e.g. name of fields it supports, name of secret it produces, etc)
-*  Where is this information located
+#### Minimum
+For a service to be bindable it **MUST** either:
+* provide a Secret that contains the binding data and reference this Secret using one of the patterns discussed [below](#pointer-to-binding-data). 
+* map its `status` properties to the corresponding binding data, using one of the patterns discussed [below](#pointer-to-binding-data).
 
-### 2.  Service Binding Schema
+To allow existing services to be bindable with zero changes, implementers of this specification (i.e. Operators that can fulfill a [service binding request](#request-service-binding)) can choose to provide a mechanism where the requester provides the mapping of existing `status` properties to their correspoding property in the [Service Binding Schema](#Service-binding-schema).  This should be a temporay state before the bindable service embraces at least the minimum requirement of this specification.
```

Comment:
@sbose78 - I've now added this note above.  Let me know if that's what you were thinking.